### PR TITLE
Example of how to set slow logs dynamically per-index

### PR DESCRIPTION
**Pull request by luizgpsantos**

### CODE REVIEW

**Comment by rjernst** [comment]
@jrodewig Can you review this please?

File: docs/reference/index-modules/slowlog.asciidoc [hunk 1]
```diff
@@ -26,7 +26,24 @@ index.search.slowlog.threshold.fetch.trace: 200ms
 index.search.slowlog.level: info
 --------------------------------------------------
 
-All of the above settings are _dynamic_ and are set per-index.
+All of the above settings are _dynamic_ and are set per-index. For example:
```
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]: Minor edit to make the `_settings` API example more explicit.
```suggestion
All of the above settings are _dynamic_ and can be set for each index using the
<<indices-update-settings, update indices settings>> API. For example: 
```

File: docs/reference/index-modules/slowlog.asciidoc [hunk 2]
```diff
@@ -83,7 +100,21 @@ index.indexing.slowlog.level: info
 index.indexing.slowlog.source: 1000
 --------------------------------------------------
 
-All of the above settings are _dynamic_ and are set per-index.
+All of the above settings are _dynamic_ and are set per-index. For example:
```
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]:
```suggestion
All of the above settings are _dynamic_ and can be set for each index using the
<<indices-update-settings, update indices settings>> API. For example: 
```

File: docs/reference/index-modules/slowlog.asciidoc [hunk 3]
```diff
@@ -26,7 +26,24 @@ index.search.slowlog.threshold.fetch.trace: 200ms
...
+    "index.search.slowlog.threshold.query.debug": "2s",
+    "index.search.slowlog.threshold.query.trace": "500ms",
+    "index.search.slowlog.threshold.fetch.warn": "1s",
+    "index.search.slowlog.threshold.fetch.info": "800ms",
+    "index.search.slowlog.threshold.fetch.debug": "500ms",
+    "index.search.slowlog.threshold.fetch.trace": "200ms",
+    "index.search.slowlog.level": "info"
+}
+--------------------------------------------------
+// CONSOLE
```
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]: This should fix the CI errors.
```suggestion
// CONSOLE
// TEST[setup:twitter]
```

File: docs/reference/index-modules/slowlog.asciidoc [hunk 4]
```diff
@@ -83,7 +100,21 @@ index.indexing.slowlog.level: info
...
+{
+    "index.indexing.slowlog.threshold.index.warn": "10s",
+    "index.indexing.slowlog.threshold.index.info": "5s",
+    "index.indexing.slowlog.threshold.index.debug": "2s",
+    "index.indexing.slowlog.threshold.index.trace": "500ms",
+    "index.indexing.slowlog.level": "info",
+    "index.indexing.slowlog.source": "1000"
+}
+--------------------------------------------------
+// CONSOLE
```
Comment (jrodewig) [inline docs/reference/index-modules/slowlog.asciidoc]:
```suggestion
// CONSOLE
// TEST[setup:twitter]
```

**Review by jrodewig — CHANGES_REQUESTED** [review]
Thanks for the contribution @luizgpsantos!

I made a few suggestions that should get this to pass the CI. As soon as those are fixed, I can help merge and backport.

**Review by jrodewig — APPROVED** [review]
LGTM. I'll merge this and backport as needed once CI passes. Thanks again @luizgpsantos!

**Comment by luizgpsantos** [comment]
Thanks @jrodewig!

**Pull request merged**